### DISCUSSION TITLE
Github.dev redirecting to github.com

### DISCUSSION THREAD

**Discussion by hnguyenHWI** [community/community#21969] [New to GitHub]
<p>This just started today, was working fine yesterday.</p>
<p>Opening any repo and pressing <code>.</code> tries to open the web editor (the loading page comes up) and then redirects to the repository view on <a href="http://github.com" rel="noopener nofollow ugc">github.com</a>.</p>
<p>The same also happens when navigating directly to the repo with <code>https://github.dev/&lt;user&gt;/&lt;repo&gt;</code>.</p>
<p>I am using Brave browser at the version:<br>
Version 1.37.116 Chromium: 100.0.4896.127 (Official Build) (64-bit)</p>
<p>I have disabled the browser’s built-in ad-blocking for both domains. I also use the extension uBlock Origin and have turned it off for both domains.</p>
<p>I also tested in Firefox, v99.0.1 64-bit, and get the same result.</p>
<p>Are there any other resources beyond the brief <a href="https://docs.github.com/en/codespaces/the-githubdev-web-based-editor#troubleshooting" rel="noopener nofollow ugc">troubleshooting section</a> in the GitHub docs that anyone knows of?</p>
<p>Thank you.</p>
<p>Edit: further info - using the Network tab in the dev tools, it looks like there’s a GET request to <code>https://api.github.com/repos/&lt;USER&gt;/&lt;REPO&gt;/git/extract-ref/HEAD</code> that fails with a 404.</p>

**Comment by hnguyenHWI** [comment, marked as answer]
<p>Posting again to update that it’s working for me now.</p>
<p>If this topic should be deleted, please go ahead and do so, or reply and let me know and I will delete.</p>

**Reply by c3-Othon** [reply]
This is an intermittent issue to me. It works some times and it redirects me some times
